jcl/src/java.base/share/classes/openj9/internal/criu/security/CRIUConfigurator.java

```diff
@@ -69,10 +71,11 @@ public static void setCRIUSecMode(Properties props) {
 	}
 
 	/**
-	 * Removes the CRIU security provider and adds the usual security providers
-	 * back.
+	 * Disables the restricted security mode. Removes the CRIU security provider
+	 * and adds the usual security providers back.
 	 */
 	public static void setCRIURestoreMode() {
+		RestrictedSecurityConfigurator.disable();
 		if (null != systemProps) {
 			Security.removeProvider("CRIUSEC");
 			// Note that CRIUSEC was set as security.provider.1 in the method
```

Comment:
> systemProps.remove("security.provider.1");

should this be adopted to the Restricted Security Mode?

Reply:
No, CRIUSEC is also `security.provider.1` in the restricted security mode, configured the `java.security` file:
```
RestrictedSecurity2.jce.provider.1 = openj9.internal.criu.CRIUSEC
```